lib/plug/adapters/cowboy/conn.ex

```diff
@@ -31,6 +31,7 @@ defmodule Plug.Adapters.Cowboy.Conn do
   end
 
   def send_resp(req, status, headers, body) do
+    status = "#{status} #{Plug.Conn.Status.string(status)}"
```

Comment:
How about another function in `Plug.Conn.Status` that generates this full status code + phrase?